packages/runtime/container-runtime/src/test/containerRuntime.spec.ts

```diff
@@ -1125,208 +1121,6 @@ describe("Runtime", () => {
-				type ContainerRuntimeWithSubmit = Omit<ContainerRuntime, "submit"> & {
-					submit(
-						containerRuntimeMessage: OutboundContainerRuntimeMessage,
-						localOpMetadata: unknown,
-						metadata: Record<string, unknown> | undefined,
-					): void;
-				};
-				const containerRuntimeWithSubmit =
-					containerRuntime as unknown as ContainerRuntimeWithSubmit;
-
-				const gcMessageWithDeprecatedCompatDetails: ContainerRuntimeGCMessage = {
-					type: ContainerMessageType.GC,
-					contents: { type: "Sweep", deletedNodeIds: [] },
-					compatDetails: { behavior: "Ignore" },
-				};
-
-				assert.doesNotThrow(
-					() =>
-						containerRuntimeWithSubmit.submit(
-							gcMessageWithDeprecatedCompatDetails,
-							undefined,
-							undefined,
-						),
-					"Cannot submit container runtime message with compatDetails",
-				);
-			});
-
-			/** Overwrites channelCollection property and exposes private submit function with modified typing */
-			function patchContainerRuntime(): Omit<ContainerRuntime, "submit"> & {
-				submit: (containerRuntimeMessage: UnknownContainerRuntimeMessage) => void;
-			} {
-				const patched = containerRuntime as unknown as Omit<
-					ContainerRuntime,
-					"submit" | "channelCollection"
-				> & {
-					submit: (containerRuntimeMessage: UnknownContainerRuntimeMessage) => void;
-					channelCollection: Partial<ChannelCollection>;
-				};
-
-				patched.channelCollection = {
-					setConnectionState: (_connected: boolean, _clientId?: string) => {},
-					// Pass data store op right back to ContainerRuntime
-					reSubmit: (type: string, envelope: any, localOpMetadata: unknown) => {
-						submitDataStoreOp(
-							containerRuntime,
-							envelope.address,
-							envelope.contents,
-							localOpMetadata,
-						);
-					},
-				} satisfies Partial<ChannelCollection>;
-
-				return patched;
-			}
-
-			it("Op with unrecognized type and 'Ignore' compat behavior is ignored by resubmit", async () => {
-				const patchedContainerRuntime = patchContainerRuntime();
-
-				changeConnectionState(patchedContainerRuntime, false, mockClientId);
-
-				submitDataStoreOp(patchedContainerRuntime, "1", "test");
-				submitDataStoreOp(patchedContainerRuntime, "2", "test");
-				patchedContainerRuntime.submit({
-					type: "FUTURE_TYPE" as any,
-					contents: "3",
-					compatDetails: { behavior: "Ignore" }, // This op should be ignored by resubmit
-				});
-				submitDataStoreOp(patchedContainerRuntime, "4", "test");
-
-				assert.strictEqual(
-					submittedOps.length,
-					0,
-					"no messages should be sent while disconnected",
-				);
-
-				// Connect, which will trigger resubmit
-				changeConnectionState(patchedContainerRuntime, true, mockClientId);
-
-				assert.strictEqual(
-					submittedOps.length,
-					3,
-					"Only 3 messages should be sent - Do not resubmit the future/unknown op",
-				);
-			});
-
-			it("Op with unrecognized type and no compat behavior causes resubmit to throw", async () => {
```

Comment:
This test should stay as well - this behavior shouldn't be changing

Reply:
Actually just keep tests for the "process" codepath, don't worry about Resubmit or applyStashed ops, those were only needed for such crazy corner cases.